src/main/scala/org/globalforestwatch/summarystats/afi/AFiAnalysis.scala

```diff
@@ -42,17 +41,48 @@ object AFiAnalysis extends SummaryAnalysis {
 //        }
 //    }.unify
 
-
     // TODO somehow convert AFiSummary to AFiData
     import spark.implicits._
 
-    val summaryDF = AFiDF.getFeatureDataFrame(summaryRDD, spark)
+    val summaryDF = AFiDF
+      .getFeatureDataFrame(summaryRDD, spark)
+      .withColumn(
```

Comment:
What case here are you dealing with by checking if null is in the GADM ID?

Reply:
I'm doing that to consistently set the column to null (specifically doing that to merge scenarios of <ABC.null.null> and <ABC.123.null>  I encountered in an example run).